src/loaders/compute_tools/tool_result_parser.py

```diff
@@ -12,6 +12,45 @@
 TOOL_GENOME_ATTRI_FILE = "genome_attribs.jsonl"
 
 
+def process_genome_attri_result(
+        output_dir: Path,
+        features: Set[str],
+        genome_id_col: str,
+        ids_to_files: Dict[str, GenomeTuple],
+        result_files: List[str],
+        check_file_exists: bool = True,
+        prefix: str = ''
+) -> None:
+    """
+    process the output files generated by the tool (checkm2, gtdb-tk, etc) to create a format suitable for
+    importing into ArangoDB
+
+    :param output_dir: the directory where the tool result files are stored
+    :param features: a list of features to be retrieved from the tool result file
+    :param genome_id_col: the column name of the genome id in the tool result file
+    :param ids_to_files: a dictionary of tool genome ids to the corresponding GenomeTuple
+    :param result_files: the list of tool result files
+    :param check_file_exists: if True, check if the tool result file exists
+    :param prefix: the prefix of the tool result file
+    """
+
+    genome_attri_docs = list()
+    for tool_file_name in result_files:
+        docs = read_genome_attri_result(
```

Comment:
I think you can just inline this method

Reply:
👍

Reply:
By inline I mean incorporate the code into the calling method and just not have a 2nd method

Reply:
👍